src/handlers/processors.ts

```diff
@@ -54,7 +53,7 @@ export const processors: Record<string, Handler> = {
   [GithubEvent.ISSUES_CLOSED]: {
     pre: [nullHandler],
     action: [issueClosedCallback],
-    post: [incentivizeCreatorComment, incentivizeComments],
+    post: [nullHandler],
```

Comment:
This seems unexpected. What's going on here?

Reply:
all incentives are now included in `issueClosedCallback`